contribs/gmf/src/directives/partials/routing.html

```diff
@@ -0,0 +1,65 @@
+<div class="gmf-routing">
+  <div id="routing-start">
+    <gmf-routing-feature
+            gmf-routing-feature-map="routeCtrl.map"
+            gmf-routing-feature-feature="routeCtrl.startFeature_"
+            gmf-routing-feature-fill-color="routeCtrl.colors['start.fill']"
+            gmf-routing-feature-stroke-color="routeCtrl.colors['start.stroke']"
+            gmf-routing-feature-on-change="routeCtrl.handleChange"
+            gmf-routing-feature-search-default-params="routeCtrl.searchDefaultParams">
+    </gmf-routing-feature>
+  </div>
+
+  <div id="routing-vias" ng-repeat="(index, via) in routeCtrl.viaArray">
+    <gmf-routing-feature
+            gmf-routing-feature-map="routeCtrl.map"
+            gmf-routing-feature-feature="via.feature"
+            gmf-routing-feature-fill-color="routeCtrl.colors['via.fill']"
+            gmf-routing-feature-stroke-color="routeCtrl.colors['via.stroke']"
+            gmf-routing-feature-on-change="routeCtrl.handleChange"
+            gmf-routing-feature-search-default-params="routeCtrl.searchDefaultParams">
+    </gmf-routing-feature>
+    <button type="button" class="btn btn-default delete-via" ng-click="routeCtrl.deleteVia(index)">
+      <span class="fa fa-trash-o"></span>
+    </button>
+  </div>
+
+  <div id="routing-destination">
+    <gmf-routing-feature
+            gmf-routing-feature-map="routeCtrl.map"
+            gmf-routing-feature-feature="routeCtrl.targetFeature_"
+            gmf-routing-feature-fill-color="routeCtrl.colors['destination.fill']"
+            gmf-routing-feature-stroke-color="routeCtrl.colors['destination.stroke']"
+            gmf-routing-feature-on-change="routeCtrl.handleChange"
+            gmf-routing-feature-search-default-params="routeCtrl.searchDefaultParams">
+    </gmf-routing-feature>
+  </div>
+
+  <button type="button" id="gmfRoutingReverseButton" class="btn btn-default" ng-click="routeCtrl.reverseRoute()">
+    <span class="fa fa-exchange"></span> {{'Reverse' | translate}}
+  </button>
+  <button type="button" id="gmfRoutingAddViaButton" class="btn btn-default" ng-click="routeCtrl.addVia()">
+    <span class="fa fa-plus"></span> {{'Add via' | translate}}
+  </button><br><br>
+
+  <div id="routingProfile" ng-if="routeCtrl.routingProfiles.length > 1">
+    <label for="gmfRoutingProfileSelect">{{'Profile' | translate}}:</label>
+    <select id="gmfRoutingProfileSelect"
+            name="gmfRoutingProfileSelect"
+            ng-model="routeCtrl.selectedRoutingProfile">
+      <option ng-repeat="profile in routeCtrl.routingProfiles" ng-value="profile">{{profile.label}}</option>
+    </select>
+  </div>
+
+  <div id="routingError" ng-hide="routeCtrl.errorMessage === ''">
+    {{routeCtrl.errorMessage}}
+  </div>
+
+  <div id="routeDuration" ng-hide="routeCtrl.routeDuration === null">
+    <strong>{{'Duration' | translate}}:</strong> {{routeCtrl.routeDuration}} {{'minutes' | translate}}
```

Comment:
instead of having:
```
<strong>{{'Duration' | translate}}:</strong>
```
you can have the simpler:
```
<strong translate>Duration:</strong>
```